utils/firestore.js

```diff
@@ -427,29 +428,56 @@ const removeParticipantsDataDestruction = async () => {
             const participant = doc.data();
             const millisecondsInDay = 24 * 60 * 60 * 1000;
             const { isIsoDate } = require('./validation');
-            const timeDiiff = isIsoDate(participant['269050420'])
+            const timeDiff = isIsoDate(participant['269050420'])
                 ? new Date().getTime() - new Date(participant['269050420']).getTime()
                 : 0
 
-            if (participant['883668444'] === 704529432 || (Math.floor(timeDiiff / millisecondsInDay) > 60)) {
+            if (participant['883668444'] === 704529432 || (Math.floor(timeDiff / millisecondsInDay) > 2)) {
```

Comment:
```suggestion
            if (participant['883668444'] === 704529432 || timeDiff > millisecondsInTwoDays)) {
```
Since milliseconds in 2 days is fixed, there's no need to do the calculation every time.

Reply:
Solved